pkgs/development/python-modules/me_cleaner/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, installShellFiles
+}:
+
+buildPythonPackage rec {
+  pname = "me_cleaner";
```

```suggestion
  pname = "me-cleaner";
```

see https://nixos.org/manual/nixpkgs/stable/#contributing

---

> change "me_cleaner" to "me-cleaner"

Done. :)

> Please add `pythonImportsCheck`.

I just learned of the pythonImportsCheck existence and to my understanding it would try to import the python modules shipped by itself to see if this already causes breakage.

The me_cleaner.py is "just" a script (that is placed in .../bin/). And there are no importable modules provided.

Output something like `find $out -type f`:
```
/share/man/man1/me_cleaner.1.gz
/lib/python3.9/site-packages/me_cleaner-1.2.dist-info/WHEEL
/lib/python3.9/site-packages/me_cleaner-1.2.dist-info/direct_url.json
/lib/python3.9/site-packages/me_cleaner-1.2.dist-info/REQUESTED
/lib/python3.9/site-packages/me_cleaner-1.2.dist-info/top_level.txt
/lib/python3.9/site-packages/me_cleaner-1.2.dist-info/COPYING
/lib/python3.9/site-packages/me_cleaner-1.2.dist-info/METADATA
/lib/python3.9/site-packages/me_cleaner-1.2.dist-info/INSTALLER
/lib/python3.9/site-packages/me_cleaner-1.2.dist-info/RECORD
/nix-support/propagated-build-inputs
/bin/.me_cleaner.py-wrapped
/bin/me_cleaner.py
```

Should I attempt to import the script itself?